### DISCUSSION TITLE
关于 Reality 实践相关的若干问题

### DISCUSSION THREAD

**Discussion by kyber1024** [XTLS/Xray-core#1932] [Q&A]
1. 部署 Reality 的服务器IP如果不为dest网站所拥有，这是否会成为一个强特征
2. Reality 为何在dest原本的ip尽可能冷门的情况下更加安全
3. Dest 本身服务的流量是有特征的，可能可以用于识别reality，[文档](https://github.com/XTLS/REALITY)也提到了要解决，模仿目前有什么具体的方向或思路吗

**Comment by nursery01** [comment]
1. 它是一個強特徵，比如網站a，它使用了AWS的伺服器，但是你卻使用Azure的伺服器，這就是一個強特徵。但是網站使用多個不同的云公司的伺服器是很正常的情況，世界上是有網站同時使用了AWS和Azure，不太可能因爲這個原因而封鎖，除非是蘋果這樣的公司。
2. 有些熱門服務的ip位址10年來都沒有發生改變，一直用同一個ip，你突然拿了個新ip套上去，看上去是很奇怪，但是一個網站是有可能啓用新的ip，不太可能因爲這個原因而封鎖，除非某些絕對固定ip的網站。
3. 你指的是具體哪方面？臨時憑證？

**Comment by kyber1024** [comment]
比如 addons.mozilla.org 和 huggingface.co 的流量大小会有明显区别，因为各自网站的内容不一样

**Comment by kyber1024** [comment]
追加一个疑问
在一个reality的实例中，节点的443端口已经被censys识别为 Amazon CloudFront Load Balancer，这符合预期。但是同时也注意到80端口并没有被开放，而一个常见的 Amazon CloudFront Load Balancer 应该开放80端口，同时服务器开放了22端口， 这也是不符合真实情况的。这些细微的差异会成为暴露风险吗，有必要做进一步的优化吗

**Reply by simplerick-simplefun** [reply]
一般都会开80，并设定为http request redirect 443。
不建议ssh使用22，容易被扫被黑。建议改用高位端口/或使用443端口通过load balancer连接。同时建议关闭密码登录，改用private key或其他方式。
